tools/run_tests/build_artifact_python.sh

```diff
@@ -43,7 +43,11 @@ GRPC_PYTHON_USE_CUSTOM_BDIST=0  \
 GRPC_PYTHON_BUILD_WITH_CYTHON=1 \
 ${SETARCH_CMD} python setup.py  \
     bdist_wheel                 \
-    sdist                       \
+    sdist
```

Comment:
Because what we are doing is hands down unorthodox, because there definitively exist more supported ways of supporting binary linux distributions (e.g. debian packages), and the issues already exist and we've already pinged them. Filing subissues flings us into the X-Y of having a particular problem that we're trying to solve and then going about it in one particular way when at any point in time we could realize there is something so much better. For example, I want to outright scrap this whole deal and try something different but I don't have the time to do so before Friday, which is a *hard deadline* for our package to be available.

Given what we have done so far, I vehemently disagree that we 'want' to get to a certain goal in an orthodox way. We simply 'want' to get to the goal for 0.13.0, and we're all signed up for cleanup afterwards.